README.md

```diff
@@ -164,6 +164,8 @@ This could take more than 10 minutes, please be patient.
 Migration Guide
 ----------------
 
+> Note: The APIs have changed since the era of `sympy.galgebra` and `brombo/galgebra`, some properties and methods are deprecated, the supported versions of Python and SymPy have also changed, please check [Changelog](https://galgebra.readthedocs.io/en/latest/changelog.html) and further update your scripts accordingly besides the following. If you encounter any problems, feel free to open an issue!
```

Comment:
Could make "open an issue" a link - remember this appears on the Sphinx docs too, where it's less obvious how to get to the issue tracker.

Reply:
```suggestion
> Note: The APIs have changed since the era of `sympy.galgebra` and `brombo/galgebra`, some properties and methods are deprecated, the supported versions of Python and SymPy have also changed, please check [Changelog](https://galgebra.readthedocs.io/en/latest/changelog.html) and further update your scripts accordingly besides the following. If you encounter any problems, feel free to [open an issue](https://github.com/pygae/galgebra/issues/new)!
```